packages/terra-form-checkbox/docs/UPGRADEGUIDE.md

```diff
@@ -1,5 +1,16 @@
 # Terra Form Checkbox Upgrade Guide
 
+## Changes from version 3 to version 4
+
+### Props
+
+#### Updated
+* `labelText` prop's type changed from node to string.
+* `legend` prop's type changed from node to string.
+
+##### Steps to Update
```

Comment:
Can we add some context as to why this change was make?

Reply:
Added in 6db571a